Sources/XCHammer/XcodeTarget.swift

```diff
@@ -963,6 +963,7 @@ public class XcodeTarget: Hashable, Equatable {
         case Watch1Extension = "com.apple.product-type.watchkit-extension"
         case Watch2Extension = "com.apple.product-type.watchkit2-extension"
         case TVAppExtension = "com.apple.product-type.tv-app-extension"
+        case IMessageExtension = "com.apple.product-type.app-extension.messages"
```

Comment:
I didn't have a chance to validate this, but if it's working for you that seems fine. Perhaps we should have a way to enumerate all of these types and make sure XCHammer has them all for a given Xcode ( or warn otherwise )